app/util/googleAnalytics.ts

```diff
@@ -0,0 +1,7 @@
+import { BYIELD_GA_EVENT_NAME, eventParam } from "~/types/googleAnalytics";
+
+export function trackEvent(eventName: BYIELD_GA_EVENT_NAME, params: eventParam) {
+	if (typeof window !== "undefined" && window.gtag) {
+		window.gtag("event", eventName, { ...params });
```

Comment:
what an can we include in params?

Reply:
it's a key value. we can push anything in it?

Reply:
let's add there two property to all events:
```
network: testnet / mainnet
prod: import.meta.env.PROD
```

Reply:
Done